RefugeeLand.Core.Api/Brokers/Storages/StorageBroker.Shelters.cs

```diff
@@ -26,5 +26,11 @@ public async ValueTask<Shelter> InsertShelterAsync(Shelter shelter)
 
             return shelterEntityEntry.Entity;
         }
```

Comment:
Hey @TylerDDev, Could you had an empty line between the two methods.

Reply:
Between the 28th and 29th line

Reply:
done